index.test.js

```diff
@@ -148,4 +149,53 @@ describe(_startCase(filename), function () {
       expect(res.redirect).toBeCalledWith('http://domain.com/_auth/login');
     });
   });
+
+  describe('init', function () {
+    const router = { use: jest.fn(), get: jest.fn() },
+      options = {
+        router,
+        site: {
+          prefix: 'foo.com',
+          protocol: 'http',
+          port: 80
+        },
+        providers: ['apikey', 'google']
+      };
+
+    it('returns if no providers are passed in', function () {
+      expect(lib({})).toEqual([]);
+    });
+
+    it('should set required middlewares', function () {
+      strategyService.createStrategy = jest.fn();
+      strategyService.addAuthRoutes = jest.fn();
+
+      lib(options);
+
+      expect(router.use).toBeCalledTimes(7);
```

Comment:
For these `toBeCalledTimes()` it might be helpful to comment what's calling it. What do you think?